VagrantProvisionCentOS7Deps.sh

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env bash
+
+set -e
+
+###################################################
+# VERY IMPORTANT: Set the $HOOT_HOME environment  #
+# variable prior to running this script if ~/hoot #
+# isn't the correct location for HOOT_HOME        #
+###################################################
+if [ -z "$HOOT_HOME" ]; then
+    HOOT_HOME=~/hoot
+fi
+echo HOOT_HOME: $HOOT_HOME
+#################################################
+
+# Common set of file versions
+source $HOOT_HOME/VagrantProvisionVars.sh
+
+if ! rpm -qa | grep -q ^yum-plugin-versionlock ; then
+    # Install the versionlock plugin version first.
+    sudo yum install -y yum-plugin-versionlock >> CentOS_upgrade.txt 2>&1
+elif [[ $(yum versionlock list | wc -l) -gt 2 ]] ; then  # versionlock list returns two lines if empty
```

Comment:
Moved everything to a new script and then modified this line.  Before it was just an `else` statement.  This could error out if `yum-plugin-versionlock` was installed but not created.  Now we actually check for a version lock existing (though trivially) before trying to delete it.